src/Hubbup.Web/Startup.cs

```diff
@@ -71,31 +80,43 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 app.UseExceptionHandler("/Error");
             }
 
-            app.UseIISPlatformHandler(options => options.AuthenticationDescriptions.Clear());
-
             app.UseStaticFiles();
 
-            app.UseCookieAuthentication(options =>
+            app.UseCookieAuthentication(new CookieAuthenticationOptions()
             {
-                options.AutomaticAuthenticate = true;
-                options.AuthenticationScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-                options.LoginPath = new PathString("/signin");
+                AutomaticAuthenticate = true,
+                AuthenticationScheme = CookieAuthenticationDefaults.AuthenticationScheme,
+                LoginPath = new PathString("/signin")
             });
 
             app.UseGitHubAuthentication(options =>
             {
                 options.ClientId = Configuration["GitHubClientId"];
                 options.ClientSecret = Configuration["GitHubClientSecret"];
                 options.Scope.Add("repo");
-                options.SaveTokensAsClaims = true;
+                options.SaveTokens = true;
+                options.AutomaticChallenge = true;
             });
 
             app.UseSession();
 
             app.UseMvc();
         }
 
-        // Entry point for the application.
-        public static void Main(string[] args) => WebApplication.Run<Startup>(args);
+        // Entry point for the application.public static void Main(string[] args)
+        public static void Main(string[] args)
+        {
+            var config = new ConfigurationBuilder().AddEnvironmentVariables().Build();
```

Comment:
For that to work it would have needed to be `.AddEnvironmentVariables("ASPNETCORE_")`, unless you were just defining `ENVIRONMENT`?